docs/topics/js/js-interop.md

```diff
@@ -202,12 +202,36 @@ fun usingAsOperator(s: Any) = s as String
 They will be compiled accordingly:
 
 ```javascript
-function usingUnsafeCast(s) {
+function usingUnsafeCast (s) {
```

Comment:
I'm not sure if this space required by JS style guide?

Reply:
https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Writing_style_guide/Code_style_guide/JavaScript#function_declarations

Reply:
Although, there are multiple opinions on that in JS ecosystem. Better to consult with someone from JS team on that. If you decide to change the existing style, please do it consistently in all examples and not in this PR.